### PR TITLE
Apply coala patches

### PR DESCRIPTION
**Pull request by manu-chroma**

### CODE REVIEW

**Comment by manu-chroma** [comment]
@jayvdb Ref to: https://travis-ci.org/manu-chroma/username-availability-checker/builds/321837314#L846
Is increasing line limit only way to get around this error?

**Comment by jayvdb** [comment]
> Is increasing line limit only way to get around this error?

no.  see https://github.com/coala/coala-bears/blob/master/bears/general/LineLengthBear.py#L23

File: tests/test_get_avatar.py [hunk 1]
```diff
@@ -1,56 +1,56 @@
...
+class TestGetAvatar(object):
+
+    @parameterized.expand(load_test_cases('with_avatar'),
+                          testcase_func_name=custom_name_func)
+    def test_with_avatar(self, website, user):
+        if not user:
+            pytest.skip('website not supported')
+        link = username_api.check_username(website, user)['avatar']
+        response = r.get(link)
+        assert(response.headers.get('content-type', '').startswith('image/') or
```
Comment (jayvdb) [inline tests/test_get_avatar.py]: Unrelated to your patch, but good to include now as it is another whitespace only change ...

`assert(..` looks like a function call.

instead, adding a space can help ensure people see a function call was not intended

`assert (..`

File: cli.py [hunk 2]
```diff
@@ -1,20 +1,21 @@
-import requests
 from username_api import check_username
 
-sites = list(map(lambda x: '{}.com'.format(x), 
-	"facebook twitter instagram github youtube soundcloud tumblr".split()))
+sites = list(map(lambda x: '{}.com'.format(x),
+                 'facebook twitter instagram \
```
[the remark this answers is not in the document]
Comment (manu-chroma) [reply cli.py]: Fixing the linting errors for now. 
I realise how badly ``cli.py`` is broken (also, we used to support ``YouTube`` back in the day ) :P 

I will open an issue to fix it separately. It is ``low priority`` and we can choose to actually fix or remove the file for later.

File: tests/test_username_api.py [hunk 3]
```diff
@@ -15,114 +14,125 @@
...
+
+    @parameterized.expand(load_availability_test_cases('available_username'),
+                          testcase_func_name=custom_name_func)
+    def test_available_username(self, website, user):
+        expected = get_expected_response(website, user, 404)
+        actual = get_response(self.app, website, user)
+
+        message = None
+        if expected != actual:
+            message = 'The provided available username ({}) ' \
```
[the remark this answers is not in the document]
Comment (manu-chroma) [reply tests/test_username_api.py]: I was very strictly trying to adhere to PEP8 limit :P

**Comment by manu-chroma** [comment]
@jayvdb :)

**Pull request merged**